std/random.d

```diff
@@ -603,7 +603,14 @@ Always `false` (random generators are infinite ranges).
  */
     enum bool empty = false;
 
-    private UIntType _x = m ? (a + c) % m : (a + c);
+    static if (m)
+    {
+        private UIntType _x = a + c % m;
```

Comment:
There is a change from `(a + c) % m` to `a + c % m`. Is this intentional?

Reply:
it was not. good catch